public/app/features/trails/AutomaticMetricQueries/query-generators/bucket/index.ts

```diff
@@ -23,28 +23,28 @@ function generator(metricParts: string[]) {
     ...common,
     variant: 'p50',
     queries: [percentileQuery(50)],
-    vizBuilder: simpleGraphBuilder,
+    vizBuilder: simpleGraphBuilder(common),
   };
 
   const breakdown: AutoQueryDef = {
     ...common,
     variant: 'p50',
     queries: [percentileQuery(50, [VAR_GROUP_BY_EXP])],
-    vizBuilder: simpleGraphBuilder,
+    vizBuilder: simpleGraphBuilder(common),
```

Comment:
```suggestion
    vizBuilder: () => simpleGraphBuilder(breakdown),
```

I was thinking more along this line

Reply:
I couldn't find a way to work around:
> Block-scoped variable 'breakdown' used before its declaration.ts(2448)

...besides using `common` instead.

Reply:
I took another look and went down this path in:
- #80170
I prefer how it makes the builder functions not require their own lambda.